### ISSUE TITLE
What do you think of Ubuntu support?

### ISSUE THREAD

**Issue by rlopzc** [nekr0z/linux-on-huawei-matebook-13-2019#6]

**Comment by nekr0z** [comment]
What works on Debian should generally work in Ubuntu, there's nothing that much Debian-specific.

**Comment by kian1991** [comment]
Well I have running Ubuntu 19.04 and it works lika charm (even touchpad - no pinch to zoom though)

Annoying Issue:

When I use GRUB as default Bootloader sometimes the notebook wont go to sleep (even on Windows 10) it gets hot in my bag an runs outta battery. 

Did you expierience something similar @nekr0z ? 

Cheers

**Comment by nekr0z** [comment]
No, I have not experienced any issues with putting the laptop to sleep. However, there is a possibility of it waking up unexpectedly, more info on this matter in #5.

**Comment by kian1991** [comment]
ah yeah looks loke the same issue thanks @nekr0z ill have a look into the logs as soon I have the opportunity which log should I have a look at?

**Comment by nekr0z** [comment]
> which log should I have a look at?

I believe Ubuntu is using systemd for all the sleep-wakeup process, so it all should be in general system log (available via `journalctl`). I don't really know if there's a way to see which ACPI event provoked wakeup, but at least you can make sure that system went to sleep properly and then woke up from ACPI event (presumably, keyboard activity).
